test/util/utils.go

```diff
@@ -131,3 +136,43 @@ func BuildKubeConfigFile(config *restclient.Config) string {
 	}
 	return ""
 }
+
+func MakePods(podName string, namespace string, memReq int64, cpuReq int64, priority int32, uid string, nodeName string) *corev1.Pod {
```

Comment:
This is mainly to make the code concise. 
for example:
```go
st.MakePod().Namespace("ns1").Name("pod1").Container(pause).Priority(priority).Node("node1").UID("pod1").ZeroTerminationGracePeriod().Req({corev1.ResourceRequirements{
		Requests: corev1.ResourceList{
			corev1.ResourceMemory: *resource.NewQuantity(100, resource.DecimalSI),
			corev1.ResourceCPU:    *resource.NewMilliQuantity(100, resource.DecimalSI),
		},
	}).Obj()
```
We can use as 
```
util.MakePods("t1-p1", "ns1", 50, 10, midPriority, "t1-p1", "")
```
If there are a lot of pods, it will be more obvious